sunpy/net/tests/test_fido.py

```diff
@@ -322,29 +323,26 @@ def test_fido_indexing(queries):
     res = Fido.search(query1 | query2)
 
     assert len(res) == 2
-    assert len(res[0]) == 1
-    assert len(res[1]) == 1
+    assert len(res[0][0]) == 1
+    assert len(res[1][0]) == 1
 
     aa = res[0, 0]
-    assert isinstance(aa, UnifiedResponse)
+    assert isinstance(aa, BaseQueryResponse)
     assert len(aa) == 1
-    assert len(aa.get_response(0)) == 1
+    assert len(aa[0]) == 1
 
     aa = res[:, 0]
     assert isinstance(aa, UnifiedResponse)
     assert len(aa) == 2
     assert len(aa.get_response(0)) == 1
 
     aa = res[0, :]
-    assert isinstance(aa, UnifiedResponse)
-    assert len(aa) == 1
+    assert isinstance(aa, BaseQueryResponse)
+    assert len(aa[0]) == 1
 
     with pytest.raises(IndexError):
         res[0, 0, 0]
 
-    with pytest.raises(IndexError):
-        res["saldkal"]
```

Comment:
What happens now?

Reply:
An empty `unifiedResponse` is returned, since no client name is "saldkal"

Reply:
Might as well repurpose that test then.

Reply:
Done.